### PR TITLE
Timm cv

### PR DESCRIPTION
**Pull request by D1MK4real**
Added timm cv-library, bug-fix in multilabel linear model, added softmax with clip to repair warning in log_loss from sklearn: "The y_pred values do not sum to one. Starting from 1.5 this will result in an error."

### CODE REVIEW

File: lightautoml/ml_algo/utils.py [hunk 1]
```diff
@@ -76,3 +81,33 @@ def tune_and_fit_predict(
             return None, None
 
     return ml_algo, preds
+
+
+class MySoftmaxClip(nn.Module):
```
Comment (DESimakov) [inline lightautoml/ml_algo/utils.py:86]: Naming.

File: pyproject.toml [hunk 2]
```diff
@@ -48,7 +48,7 @@ catboost = ">=0.26.1"
 optuna = "*"
 torch = [
     {platform = "win32", python = "3.6.1", version = "1.7.0"},
-    {version = ">=1.0.0"}
+    {version = "<=2.0.0"}
```
Comment (DESimakov) [inline pyproject.toml:51]: versions min and max

**Review by DESimakov — CHANGES_REQUESTED** [review]

**Review by DESimakov — APPROVED** [review]
